app/views/charts/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Charts</h1>
+
+<table>
+  <thead>
```

Comment:
since I've implemented this:
```ruby
class Chart < ActiveRecord::Base
 validates :data, presence: true
end
```
this spec does not pass anymore
charts_controller_spec
```ruby
      it 'creates a new Chart' do
        expect {
          post :create, chart: valid_attributes
        }.to change(Chart, :count).by(1)
      end
```
and I have no Idea.... because manually I can create entries

```
Failures:

  1) ChartsController POST #create with valid params creates a new Chart
     Failure/Error:
       expect {
         post :create, chart: valid_attributes
       }.to change(Chart, :count).by(1)

       expected #count to have changed by 1, but was changed by 0
     # ./spec/controllers/charts_controller_spec.rb:70:in `block (4 levels) in <top (required)>'

Finished in 0.63365 seconds (files took 8.88 seconds to load)
1 example, 1 failure
```